### DISCUSSION TITLE
random names contain hurtful terms

### DISCUSSION THREAD

**Discussion by TomatoLoli** [dyc3/opentogethertube#1819] [Q&A] [answered]
A buddy of mine got the name 'homeless_ivory_antelope'. I'm sure its not intentional, but wanted to bring attention to it and was curious if anyone else saw stuff like that generated?

Thanks.

**Comment by dyc3** [comment, marked as answer]
Sorry about that, it's certainly not intentional. OTT uses a library called [`unique-names-generator`](https://www.npmjs.com/package/unique-names-generator) to generate the random names. 

Just so I can understand better and make more informed decisions, which part of the name is offensive?

**Reply by TomatoLoli** [reply]
Homeless in this case

**Reply by dyc3** [reply]
Could you elaborate why this term is offensive?
